common/sysinfo_linux.go

```diff
@@ -71,7 +71,7 @@ func GetMemAvailable() (int64, error) {
 	}
 
 	if err = cmd.Wait(); err != nil {
-		err := fmt.Errorf("GetMemInfo failed with error: %v", err)
+		err := fmt.Errorf("GetMemAvailable failed with error: %v", err)
```

Comment:
update all other places in this func..
sorry, github won't let me comment there

Reply:
I should be sorry i missed this.